squidpy/_docs.py

```diff
@@ -141,6 +141,19 @@ def decorator2(obj: Any) -> Any:
 layer_added
     Layer of new image layer to add into ``img`` object."""
 
+_ripley_stat_returns = """\
+If ``copy = True``, returns a :class:`dict` with following keys:
+
+    - `'{mode}_stat'` - :class:`pandas.DataFrame` containing the statistics of choice for the real observations.
+    - `'sims_stat'` - :class:`pandas.DataFrame` containing the statistics of choice for the simulations.
+    - `'bins'` - :class:`numpy.ndarray` containing the support.
+    - `'pvalues'` - :class:`numpy.ndarray` containing the pvalues for the statistics of interest.
+
+Otherwise, modifies the ``adata`` object with the following key:
+
+    - :attr:`anndata.AnnData.uns` ``['{key_added}']`` - the above mentioned :class:`dict`.
+
+Statistics and pvalues are computed for each cluster `'{cluster_key}'` separately."""
```

Comment:
I'd either rop the `'{cluster_key}'` or use:
... for each cluster in :attr:`anndata.AnnData.obs` ``['{cluster_key}']`` ....